easybuild/easyconfigs/i/impi/impi-2017.0.98-iccifort-2017.0.35-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,35 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'impi'
+version = '2017.0.98'
```

Comment:
@wpoely86 upstream version number for latest `impi` is `2017.0.098`?

cc @akesandgren